### DISCUSSION TITLE
trend line regression line for time series

### DISCUSSION THREAD

**Discussion by hafeja** [grafana/grafana#65328] [Dashboards, Panels & Visualizations]
As user of Grafana's  [time series](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/time-series/) plugin, I would like to request the feature of a **trend/regression line** in order to show the linear correlation.

The dashboard creator should be able to select the type of regression line, similar to the [scatter panel](https://grafana.com/grafana/plugins/michaeldmoore-scatter-panel/):
![image](https://user-images.githubusercontent.com/59864250/227866151-155ce9c4-ce27-4592-97d0-ab57ddc4f241.png)

Related post in Grafana community: https://community.grafana.com/t/is-there-any-way-to-add-a-trend-line-to-a-graph/25876

**Comment by hafeja** [comment]
Hello @ryantxu, what is your opinion about this feature request?

**Comment by bobcamp** [comment]
Hi

If somebody is going to dive in and put "fitting" into the arsenal of tools, they might as well allow more complex curves to be fitted. The fitting code isn't all that crazy. The "big deal" is getting it all into the UI so that folks can use it. There are a lot of data situations that beg the question "are we getting better or worse?". A trend line (or curve) is a really good way to answer that on a  (typically) chaotic data set. 

Bob

**Comment by gelicia** [comment]
Hello! We want to let you all know that we are moving to Github Issues for feature requests. You can read more about the move [here](https://github.com/grafana/grafana/discussions/73424).

This discussion was converted into an issue, which is available [here](https://github.com/grafana/grafana/issues/73441).
